components/actionbutton/index.css

```diff
@@ -10,7 +10,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../commons/basebutton-coretokens.css";
+@import "../commons/basebutton.css";
```

Comment:
All references to the original commons/basebutton file which used the legacy tokens are gone now so we don't need to continue hosting both versions.